Makefile

```diff
@@ -166,13 +166,13 @@ install: build doc sdk doc-json
 	dune install --destdir=$(DESTDIR) --prefix=$(PREFIX) --libdir=$(LIBDIR) --mandir=$(MANDIR) \
 		xapi-client xapi-schema xapi-consts xapi-cli-protocol xapi-datamodel xapi-types \
```

Comment:
Changing the list to have a package per file should make it easier to maintain. Also note that the install and uninstall lists are different as well.

Reply:
Maybe build the list of packages in a variable constructed incrementally with `+=` and using it for install and uninstall.